pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -217,6 +216,32 @@ stdenv.mkDerivation rec {
   '';
   preBuild = "cd build";
 
+  doCheck = true;
+  checkInputs = [ socat ];
+  preCheck = ''
+    # time limits are a little meagre for a build machine that's
+    # potentially under load.
+    substituteInPlace ../tests/unit/meson.build \
+      --replace 'timeout: slow_tests' 'timeout: 50 * slow_tests'
+    substituteInPlace ../tests/qtest/meson.build \
+      --replace 'timeout: slow_qtests' 'timeout: 50 * slow_qtests'
```

Comment:
Might be worth seeing if upstream will increase their timeouts — I imagine fewer spurious failures due to timeouts would make development easier.

Reply:
Might do yes.